crypto/fipsmodule/evp/evp.c

```diff
@@ -425,19 +422,9 @@ EC_KEY *EVP_PKEY_get1_EC_KEY(const EVP_PKEY *pkey) {
   return ec_key;
 }
 
-DH *EVP_PKEY_get0_DH(const EVP_PKEY *pkey) {
-  SET_DIT_AUTO_DISABLE;
-  return NULL;
-}
-
-DH *EVP_PKEY_get1_DH(const EVP_PKEY *pkey) {
-  SET_DIT_AUTO_DISABLE;
```

Comment:
hm, this isn't touching secret material. we didn't need this call here previously, did we?

Reply:
Really good point, let me ask internally about the justification behind this and if I should add this back.